docs/src/reference/the-traversal.asciidoc

```diff
@@ -1084,6 +1084,42 @@ g.V().identity()
 
 link:++http://tinkerpop.apache.org/javadocs/x.y.z/core/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.html#identity--++[`identity()`]
 
+[[index-step]]
```

Comment:
I think you should have something in upgrade docs to introduce this feature so that folks know it is there.